### PR TITLE
vaddr_t -> ptraddr_t

### PR DESCRIPTION
**Pull request by brooksdavis**

### CODE REVIEW

**Comment by jrtc27** [comment]
This was only ever built as part of cheribsdtest for MIPS, FWIW, so we should probably just nuke it from the cheribsd tree. The tests were of mixed quality and should be superseded by cheribsdtest's own.

File: clang-hybrid/memcpy.c [hunk 1]
```diff
@@ -40,7 +40,7 @@ __FBSDID("$FreeBSD$");
 #include <stdint.h>
 #else
 #ifndef _VADDR_T_DECLARED
```
Comment (jrtc27) [inline clang-hybrid/memcpy.c]: Macro needs changing surely?

Comment (brooksdavis) [reply clang-hybrid/memcpy.c]: Oops, I did that in CheriBSD and failed to bring it over.

**Comment by brooksdavis** [comment]
Good point, we should probably just GC the subrepo in CheriBSD and archive this repo.

**Comment by jrtc27** [comment]
Yeah, ptraddr_t-ify it then archive I think

**Comment by arichardson** [comment]
They was/is also a subrepo in the MIPS baremetal test suite but that is also dead code now I guess

**Pull request merged**